src/components/views/settings/ChangeAvatar.js

```diff
@@ -24,11 +24,21 @@ module.exports = React.createClass({
     propTypes: {
         initialAvatarUrl: React.PropTypes.string,
         room: React.PropTypes.object,
+
+        // if set, set initialAvatarUrl to the current avatar, if it has one
+        groupId: React.PropTypes.string,
         // if false, you need to call changeAvatar.onFileSelected yourself.
         showUploadSection: React.PropTypes.bool,
         width: React.PropTypes.number,
         height: React.PropTypes.number,
-        className: React.PropTypes.string
+        className: React.PropTypes.string,
+
+        // If true, set the room / user avatar once the image is uploaded.
+        // Ignored for groups.
+        setAvatar: React.PropTypes.bool,
+
+        // Called after the avatar is uploaded
+        onAvatar: React.PropTypes.func,
```

Comment:
`onUploadComplete` maybe? And it'd be nice to have a comment saying what arguments this function is called with